packages/framework/src/Resources/views/Components/Collector/shopsysFramework.html.twig

```diff
@@ -37,7 +37,11 @@
         <div class="sf-toolbar-info-group">
             <div class="sf-toolbar-info-piece">
                 <b>Resources</b>
-                <span><a href="https://github.com/shopsys/shopsys/blob/{{collector.docsVersion}}/docs/index.md" target="_blank" rel="help">Documentation</a></span>
+                {% if collector.docsVersion is not empty %}
```

Comment:
For current master branch is version string 8.1.0-dev and that means `collector.docsVersion` will never be empty.

Reply:
It was used in case we could not parse a version but it is not necessary any more.